lib/graphql/enum_type.rb

```diff
@@ -83,7 +83,8 @@ def to_s
     #
     # Created with {EnumType#value}
     class EnumValue
-      attr_reader :name, :description, :deprecation_reason, :value
+      attr_reader :name, :value
```

Comment:
I don't have a need for it, but if you think it's all right, I'll make the change here.